instruqt/proxy/03-app-install/assignment.md

```diff
@@ -0,0 +1,74 @@
+---
+slug: app-install
+id: zfxhfqtljsrb
+type: challenge
+title: Kots application install with proxy config
+teaser: Install kots application via proxy server
+notes:
+- type: text
+  contents: |
+    # Install kots application using a proxy server
+tabs:
+- title: Host
+  type: terminal
+  hostname: isolated-host
+- title: Vendor
+  type: website
+  url: https://vendor.replicated.com
+  new_window: true
+- title: KotsAdm
+  type: website
+  url: http://isolated-host.${_SANDBOX_ID}.instruqt.io:8800
+  new_window: true
+difficulty: basic
+timelimit: 1800
+---
+
+💡 Shell
+=========
+
+Install sample application with proxy config in place.
+
+The license file for the application has already been downloaded in the home directory.
+
+Even thought the kurl installer spec included kotsadm which can be see running the application installation can be completed by the command line with the kot install command using the license.
+
+Install using kots cli including license file to upload automatically vs uploading via kotsadm web UI
+```
+kubectl kots install proxy-$INSTRUQT_PARTICIPANT_ID --skip-preflights --license-file ~/license.yaml --shared-password mytestapp --namespace default --no-port-forward
+```
+
+Text access to kotsapp via the KotsAdm tab..
```

Comment:
What do you mean with that?

Reply:
:) good catch, ty, should read "Test" not "Text"

Reply:
Reading it back, could also be a more complete sentence.